internal/wasi/wasi_test.go

```diff
@@ -1128,9 +1129,9 @@ func TestSnapshotPreview1_FdWrite(t *testing.T) {
 			errno := tc.fdWrite()(ctx(mem), fd, iovs, iovsCount, resultSize)
 			require.Equal(t, wasi.ErrnoSuccess, errno)
 
-			actual, ok := mem.Read(0, uint32(len(expectedMemory)))
+			actualMemory, ok := mem.Read(0, uint32(len(expectedMemory)))
```

Comment:
nit unless there are multiple "actual" we don't need to suffix it. you'll notice 'actual' is a common name in assertions regardless of the type it points to

same for expected 

ex.
```
func Equal(t TestingT, expected interface{}, actual interface{}
```

Reply:
my guess is you did this because above we say `expectedMemory` and `initialMemory` etc. if we choose to change actual to actualMemory, do it everywhere. doing something in only a couple places out of the dozens in the same file leads to mental glitches in future maintenance.

Reply:
Thanks, that totally makes sense and your guess is right. Let me fix it.